<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Modul</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="Number_modul">
      <property name="text">
       <string>487</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Calculator</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="3" column="1">
         <widget class="QLineEdit" name="result"/>
        </item>
        <item row="2" column="1">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="2">
           <widget class="QPushButton" name="moduloDivision">
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="plus">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="multiplication">
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="minus">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>b = </string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="getInverse_a">
            <property name="text">
             <string>Get Inverse of a</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="Number_a">
            <property name="inputMask">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>a = </string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="getModul_b">
            <property name="text">
             <string>Get Modul b</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="Number_b"/>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="getInverse_b">
            <property name="text">
             <string>Get Inverse of b</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="getModul_a">
            <property name="text">
             <string>Get Modul a</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Result</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Порядок елемент групи</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Елемент групи: </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>60</y>
          <width>601</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>110</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="result_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>410</y>
          <width>641</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>410</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Result</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Степінь точки</string>
       </attribute>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>120</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>y</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>30</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>a</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>100</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>b</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>30</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>100</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>220</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>220</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Степінь</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>390</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Result</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="result_x">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>380</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="result_y">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>410</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>380</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>410</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>y</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Ейлер Кармайкл</string>
       </attribute>
       <widget class="QLineEdit" name="euler_number">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="euler_button">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>240</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ейлер</string>
        </property>
       </widget>
       <widget class="QPushButton" name="karmikle_button">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>240</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Кармайкл</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="euler_result">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>350</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>350</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Result</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Страница</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Страница</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Страница</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Страница</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Страница</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Страница</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
